src/main/java/com/palantir/gradle/revapi/RevapiJavaTask.java

```diff
@@ -67,8 +71,18 @@
         return newApiJars;
     }
 
+    public final Property<GroupAndName> getOldGroupAndName() {
```

Comment:
you need to mark these with `@Input` (`getOldVersion` too)

Reply:
ah I see that one of the implementations has `getOldGroupNameVersion` which is marked as input.

Why don't we delete these two then and pull up `getOldGroupNameVersion` into this abstract class, and extract the group+name and version respectively from that, so there's a single source of truth?

Reply:
I'm wasn't planning to mark anything as `@Input` until it actually caches properly :p